src/common/client-app.ts

```diff
@@ -65,19 +73,12 @@ class App {
 
   onInterval= () => {
     var current = this.boardVM.toPlain();
-    var myChanges = rfc6902.createPatch(this.shadowServer, current);
+    var myChanges = rfc6902.createPatch(this.shadowClient, current);

```

Comment:
This change is not related with the refactor right? only curiosity, but what does this change means?